src/app/components/publiclists/teamlist/teamcard/teamcard.component.ts

```diff
@@ -85,6 +112,12 @@ export class TeamcardComponent implements OnInit, OnChanges {
    */
   ngOnInit() {
     this.updateView();
+    this.authServiceSubscription = this.authService.change.subscribe((authState) => {
+      this.authState = authState;
+      if (this.authState.isLoggedIn) {
+        this.authState = authState;
```

Comment:
I didn't get this thing, the way it it written currently I don't find any use of `if` condition here cause in line 116 it is already assigning to the variable. @Kajol-Kumari

Reply:
yes @Sanji515 I had applied this check for getting username but now i have replaced it with `authstate` variable so yeah we don't need this condition check anymore.